qiskit/schemas/qobj_schema.json

```diff
@@ -153,6 +154,30 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "delay"
+                            ]
+                        },
+                        "val": {
```

Comment:
What is "val"? I also do not see `duration` or `t0` keys defined?

Reply:
fixed in [44303cd](https://github.com/Qiskit/qiskit-terra/pull/4212/commits/44303cd75e146cf0e793b61c636d0f77f93aed17)